### ISSUE TITLE
The CEN Tables do not use th 

### ISSUE THREAD

**Issue by Captainkirkdawson** [FreeUKGen/FreeCENMigration#776]
The cen tables are all constructed with td elements rather than th for column headings. This is likely to cause accessibility issues.

Agreed need to fix this, and also thead and tbody should be used appropriately.

E0 U3 S 9 C5: P17
